jpo-asn-j2735-2024/src/main/java/us/dot/its/jpo/asn/j2735/r2024/CooperativeControlMessage/CCMpartIIExtension.java

```diff
@@ -0,0 +1,69 @@
+/*==============================================================================
+ *
+ * This source file was generated by a tool.
+ * Beware manual edits might be overwritten in future releases.
+ * asn1jvm v1.0-SNAPSHOT
+ *
+ *------------------------------------------------------------------------------
+ * Copyright 2024 USDOT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *============================================================================*/
+
+package us.dot.its.jpo.asn.j2735.r2024.CooperativeControlMessage;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import us.dot.its.jpo.asn.runtime.annotations.Asn1ParameterizedTypes;
+import us.dot.its.jpo.asn.runtime.annotations.Asn1ParameterizedTypes.IdType;
+import us.dot.its.jpo.asn.runtime.serialization.ParameterizedTypeDeserializer;
+
+@JsonInclude(Include.NON_NULL)
+@Asn1ParameterizedTypes(
+    idProperty = "partII-Id",
+    idType = IdType.INTEGER,
+    valueProperty = "partII-Value",
+    value = {
+      @Asn1ParameterizedTypes.Type(
+          value =
+              us.dot.its.jpo.asn.j2735.r2024.Common.VehicleSafetyExtensionsCCMpartIIExtension.class,
+          intId = 0),
+      @Asn1ParameterizedTypes.Type(
+          value =
+              us.dot.its.jpo.asn.j2735.r2024.CooperativeControlMessage
+                  .LightVehicleCCMExtensionsCCMpartIIExtension.class,
+          intId = 1),
+      @Asn1ParameterizedTypes.Type(
+          value =
+              us.dot.its.jpo.asn.j2735.r2024.CooperativeControlMessage
+                  .HeavyTruckCCMExtensionsCCMpartIIExtension.class,
+          intId = 2)
+    })
+@JsonDeserialize(
+    using =
+        us.dot.its.jpo.asn.j2735.r2024.CooperativeControlMessage.CCMpartIIExtension
+            .CCMpartIIExtensionDeserializer.class)
+public abstract class CCMpartIIExtension<TValue> extends CCMPartIIcontent<TValue> {
+
+  public CCMpartIIExtension(int id, String name) {
+    super(id, name);
+  }
+
+  public static class CCMpartIIExtensionDeserializer

```

Comment:
```suggestion
  @java.lang.SuppressWarnings("rawtypes")
  public static class CCMpartIIExtensionDeserializer
```

suggestion: address warning on raw types

Reply:
Regenerated the pojos to suppress that warning and for all the other implementations of ParameterizedTypeDeserializer. see commit:
https://github.com/CDOT-CV/jpo-asn-pojos/pull/17/commits/06bb56239b4fb45772a9908b98cf4e8e383c6136